README.md

```diff
@@ -330,6 +330,32 @@ prompt = "Face of a yellow cat, high resolution, sitting on a park bench"
 image = pipe(prompt=prompt, image=init_image, mask_image=mask_image).images[0]
 ```
 
+### Weight Reuse
+
+It's possible to reuse weights between pipelines. For example, if you wanted to use a StableDiffusionPipeline and a StableDiffusionImg2ImgPipeline, you could initialize the weights for the first and apply them to the latter:
+```python
+# make sure you're logged in with `huggingface-cli login`
+from diffusers import StableDiffusionPipeline, StableDiffusionImg2ImgPipeline
+
+SD = StableDiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5")
```

Comment:
nit: all uppercase var names don't match the style of the rest of the code or READMEs, in my opinion. Would it be possible to find some lower case names instead? :)

Reply:
done